subsys/bluetooth/host/sdp.c

```diff
@@ -1282,9 +1352,10 @@ static uint16_t sdp_svc_search_att_req(struct bt_sdp *sdp, struct net_buf *buf,
 			dry_run = true;
 
 			/* Add continuation state */
-			net_buf_add_u8(rsp_buf, 2);
+			net_buf_add_u8(rsp_buf, SDP_SSA_CONT_STATE_SIZE);
 			net_buf_add_u8(rsp_buf, state.current_svc);
 			net_buf_add_u8(rsp_buf, state.last_att);
+			net_buf_add_le16(rsp_buf, state.last_att_offset);
```

Comment:
Isn't SDP a big-endian protocol?

Reply:
It has no relation with the SDP endian. this field is the data of ContinuationState. A send this field data to B, and B send it back without any parsing, then B parse this data and know the data offset.